chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html

```diff
@@ -20,18 +20,19 @@
   <div class="col-md-4 middlebdr no-pad">
     <a [routerLink]="['/demo']" [queryParams]="{showSubNavBar: false}">
       <div class="grediant">
-        <img src="assets/images/screens-01.png">
+        <img src="src/app/assets/images/screens-01.png">
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted to assets/images.